ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-sdk-e2e-tests.robot

```diff
@@ -0,0 +1,57 @@
+*** Settings ***
+Documentation     Codeflare-sdk E2E tests - https://github.com/project-codeflare/codeflare-sdk/tree/main/tests/e2e
+Suite Setup       Prepare Codeflare-sdk E2E Test Suite
+Suite Teardown    Teardown Codeflare-sdk E2E Test Suite
+Library           OperatingSystem
+Library           Process
+Resource          ../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
+
+
+*** Variables ***
+${CODEFLARE-SDK_DIR}                codeflare-sdk
+${CODEFLARE-SDK_REPO_URL}           %{CODEFLARE-SDK_REPO_URL=https://github.com/project-codeflare/codeflare-sdk.git}
+${CODEFLARE-SDK_REPO_BRANCH}        %{CODEFLARE-SDK_REPO_BRANCH=main}
+
+
+*** Test Cases ***
+Run TestMNISTRayClusterSDK test
+    [Documentation]    Run Go E2E test: TestMNISTRayClusterSDK
+    [Tags]  ODS-2544
+    ...     Sanity
+    ...     DistributedWorkloads
+    ...     Codeflare-sdk
+    Run Codeflare-sdk E2E Test    TestMNISTRayClusterSDK
+
+
+*** Keywords ***
+Prepare Codeflare-sdk E2E Test Suite
+    [Documentation]    Prepare codeflare-sdk E2E Test Suite
```

Comment:
All Tests Suites should call `RHOSi Setup` in the Suite Setup (example in ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/130__rhods_operator.robot)

Reply:
Added